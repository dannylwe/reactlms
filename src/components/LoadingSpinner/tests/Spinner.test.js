import React from "react";
import renderer from "react-test-renderer";
import Spinner from "../Spinner";

describe("Spinner", () => {
  it("should render without failing", () => {
    const wrapper = renderer.create(<Spinner />);
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
