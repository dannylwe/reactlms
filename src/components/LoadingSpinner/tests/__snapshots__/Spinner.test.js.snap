// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should render without failing 1`] = `
<i
  className="fa fa-spinner fa-spin"
/>
`;
